packages/webapp/pages/squads/[handle]/index.tsx

```diff
@@ -123,18 +134,40 @@ const SquadPage = ({ handle }: SourcePageProps): ReactElement => {
     });
   }, [isForbidden, squadId, handle, trackEvent]);
 
-  if ((isFallback || isLoading) && !isFetched) return <SquadLoading />;
+  const seoData = squad || initialData;
+  const seo = !!seoData && (
+    <NextSeo
+      title={
+        referringUser
+          ? `${referringUser.name} invited you to ${seoData.name}`
+          : `${seoData.name} posts on daily.dev`
+      }
+      description={seoData.description}
+      openGraph={{
+        images: seoData.image ? [{ url: seoData.image }] : undefined,
```

Comment:
Should we maybe have a fallback image for this?

Reply:
Also perhaps in place for token page (not sure)

Reply:
Product decided we should use default image as fallback for now. Added it here for squad page and token page 1091a6b05797524dc6b49245d072bdad06fcfbdf